packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -148,9 +120,82 @@ describe('Deposit ERC20 Token', () => {
   context('user has 0 L1-ERC20 and is on L2', () => {})
   // TODO
   context('user has 0 L1-ERC20 and is on wrong chain', () => {})
-  // TODO
+
   context(
     'user has some ERC-20 tokens which require token approval permission and is on L1',
-    () => {}
+    () => {
+      // log in to metamask before deposit
+      before(() => {
+        cy.login('L1')
+      })
+      after(() => {
+        // after all assertions are executed, logout and reset the account
+        cy.logout()
+      })
+
+      it('should add ERC20 token correctly', () => {
+        addErc20Token({
+          address: unbridgedERC20Token,
+          name: 'L1CustomToken',
+          symbol: 'LCT'
+        })
+        cy.findByPlaceholderText('Enter amount')
+          .type('0.00012', { scrollBehavior: false })
+          .then(() => {
+            cy.findByRole('button', {
+              name: /Move funds to Arbitrum Goerli/i
+            })
+              .should('be.visible')
+              .should('be.enabled')
+              .click({ scrollBehavior: false })
+            // New dialog to approve the token
+            cy.findByText('New Token Detected').should('be.visible')
+            cy.findByRole('button', { name: 'Continue' })
```

Comment:
I don't think it is selecting the checkbox before proceeding. Hence the Continue button is not getting activated.
<img width="973" alt="image" src="https://user-images.githubusercontent.com/7558499/200844575-5c1fd46a-86cb-443e-a97b-712577f0be20.png">

Reply:
As a result - 
<img width="787" alt="image" src="https://user-images.githubusercontent.com/7558499/200845108-0d51981b-ab20-4846-b428-4c7ec9ce0604.png">